spark-submit-flamegraph

```diff
@@ -237,6 +237,11 @@ main() {
   influx_uri=http://${local_ip}:${influx_http_port}
   flamegraph_title="Spark Application"
 
+  if [ -n "${WORKER_NODE}" ]; then
```

Comment:
Maybe call this explicitly, like "ONLY_INSTALL_DEPS", which will be self explanatory?

Reply:
Good idea!